From 562e7e4ba53a52e4ef7c663c078be080c49acb05 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zolt=C3=A1n=20B=C3=B6sz=C3=B6rm=C3=A9nyi?=
 <zboszor@gmail.com>
Date: Mon, 1 Dec 2025 13:55:44 +0100
Subject: [PATCH] SuperLU: Don't redeclare zcheck_tempv()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

and neither scheck_tempv().

These are declared in slu_zdefs.h

Signed-off-by: Zoltán Böszörményi <zboszor@gmail.com>
Upstream-Status: Backport
---
 scipy/sparse/linalg/_dsolve/SuperLU/SRC/spanel_bmod.c | 1 -
 scipy/sparse/linalg/_dsolve/SuperLU/SRC/zpanel_bmod.c | 1 -
 2 files changed, 2 deletions(-)

diff --git a/scipy/sparse/linalg/_dsolve/SuperLU/SRC/spanel_bmod.c b/scipy/sparse/linalg/_dsolve/SuperLU/SRC/spanel_bmod.c
index 8f18fb80b4..dd002bc9e8 100644
--- a/scipy/sparse/linalg/_dsolve/SuperLU/SRC/spanel_bmod.c
+++ b/scipy/sparse/linalg/_dsolve/SuperLU/SRC/spanel_bmod.c
@@ -43,7 +43,6 @@ at the top-level directory.
  */
 void slsolve(int, int, float *, float *);
 void smatvec(int, int, int, float *, float *, float *);
-extern void scheck_tempv();
 
 /*! \brief
  *
diff --git a/scipy/sparse/linalg/_dsolve/SuperLU/SRC/zpanel_bmod.c b/scipy/sparse/linalg/_dsolve/SuperLU/SRC/zpanel_bmod.c
index 16932b0471..ee5fbece53 100644
--- a/scipy/sparse/linalg/_dsolve/SuperLU/SRC/zpanel_bmod.c
+++ b/scipy/sparse/linalg/_dsolve/SuperLU/SRC/zpanel_bmod.c
@@ -43,7 +43,6 @@ at the top-level directory.
  */
 void zlsolve(int, int, doublecomplex *, doublecomplex *);
 void zmatvec(int, int, int, doublecomplex *, doublecomplex *, doublecomplex *);
-extern void zcheck_tempv();
 
 /*! \brief
  *
-- 
2.52.0

